docker/usr/bin/entrypoint

```diff
@@ -22,6 +30,13 @@ for FOLDER in /data/gitea/conf /data/gitea/log /data/git /data/ssh; do
     mkdir -p ${FOLDER}
 done
 
+if [ -f /data/gitea/conf/app.ini ]; then
+    echo "Found app.ini config file, migrating database"
+    chmod 644 /data/gitea/conf/app.ini
```

Comment:
This will allow anyone with access to the server to read the `app.ini`, exposing any credentials (database, metrics, smtp).

Reply:
Hmm... I guess you're right. I guess this should be `600`. In some ways this is not so bad because this for docker - you shouldn't have other users on it in any case, that's not the docker way.

Reply:
Is there any reason to change the file permissions?

Reply:
Possibly not. If you've not set the `INSTALL_LOCK` setting in your `app.ini` and you've got the wrong mode then you won't be able to use `/install`. However I guess we shouldn't be being preventing people from shooting themselves in the foot like that. Pop a PR on if you're able otherwise I can when I'm next at a dev box. @techknowlogick do you agree that perhaps we shouldn't changing the mode here?

Reply:
@pciavald What was your reasoning for changing the mode?